sql-plugin/src/main/312db/scala/com/nvidia/spark/rapids/shims/AQEUtils.scala

```diff
@@ -28,6 +28,5 @@ object AQEUtils {
     ShuffleQueryStageExec(sqse.id, reusedExchange, sqse.originalPlan)
   }
 
-  // currently we don't support AQE on Databricks
-  def isAdaptiveExecutionSupportedInSparkVersion: Boolean = false
+  def isAdaptiveExecutionSupportedInSparkVersion: Boolean = true
```

Comment:
we explicitly put this in to work around an issue with delta lake on Databricks in 10.4, sync with me so we can make sure we test that issue.  ie this is used in isDeltaLakeMetadataQuery, fi all of these are true now we may be able to remove that code.

Reply:
Can you file a follow on issue to test and remove this workaround?

Reply:
Filed https://github.com/NVIDIA/spark-rapids/issues/6539